internal/command/deploy/deploy.go

```diff
@@ -287,6 +288,7 @@ func determineImage(ctx context.Context, appConfig *app.Config) (img *imgsrc.Dep
 		Publish:         flag.GetBool(ctx, "push") || !flag.GetBuildOnly(ctx),
 		ImageLabel:      flag.GetString(ctx, "image-label"),
 		NoCache:         flag.GetBool(ctx, "no-cache"),
+		DockerHost:      flag.GetString(ctx, "docker-host"),
```

Comment:
it looks like this is a buildpacks only option. how about:

```suggestion
		BuildpacksDockerHost:      flag.GetString(ctx, "buildpacks-docker-host"),
```